### PR TITLE
chore: Add make_subject_uri function for creating subject claim URIs

### PR DESCRIPTION
**Pull request by 0marSalah**

### CODE REVIEW

File: claims_to_nodes/pipe.py [hunk 1]
```diff
@@ -92,14 +92,14 @@ def process_claim(raw_claim):
                 source_node = get_or_create_node(raw_claim['sourceURI'], raw_claim)
 
             # Create the claim node
-            claim_uri = raw_claim['claimAddress'] or 'https://live.linkedtrust.us/claims/{}'.format(raw_claim['id'])
+            claim_uri = raw_claim['claimAddress'] or make_subject_uri(raw_claim['id'])
```
Comment (gvelez17) [inline claims_to_nodes/pipe.py:95]: if we're going to do this we could also include the logic about claimAddress, and just do  `make_subject_uri(raw_claim)`  :-)

Comment (0marSalah) [reply claims_to_nodes/pipe.py:95]: yeah, good point :"

**Comment by 0marSalah** [comment]
@gvelez17 I did your comment

can I merge now

**Review by gvelez17 — APPROVED** [review]

**Comment by gvelez17** [comment]
yes please - @0marSalah please ping me in slack if i miss a review, or go ahead if you are confident and merge and ping me after!  we should never slow down a merge by more than 1 biz day

**Pull request merged**